project/SparkBuild.scala

```diff
@@ -765,16 +750,6 @@ object CopyDependencies {
 
 }
 
-object Java8TestSettings {
-  import BuildCommons._
-
-  lazy val settings = Seq(
-    javacJVMVersion := "1.8",
-    // Targeting Java 8 bytecode is only supported in Scala 2.11.4 and higher:
-    scalacJVMVersion := (if (System.getProperty("scala-2.10") == "true") "1.7" else "1.8")
```

Comment:
I think this was the config for the java8-tests module that's gone now, so it's not needed?

I think that comment meant 2._10_.4 but I need to check 2.10 compatibility before going much further.